### PR TITLE
TR001 -  Header Page

### PR DESCRIPTION
**Pull request by muruganvenkatesan**
created header page to show logo,project name,hamburger icon and current logged in user name  for thervu app

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.9.10",
```
Comment (Keerthu94) [inline package.json:7]: Why added material-ui when we decided not to use? Is this also needed for icons?

File: src/header/header.component.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './header.css';
+import Assignment from '@material-ui/icons/Assignment';
+import Hamburger from '@material-ui/icons/Menu';
+
+const header = props =>{
+  
+  let projectNameClassName = ["project-name-content"];
```
Comment (Keerthu94) [inline src/header/header.component.js]: Why didnt you use const?

File: src/header/header.component.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './header.css';
+import Assignment from '@material-ui/icons/Assignment';
+import Hamburger from '@material-ui/icons/Menu';
+
+const header = props =>{
```
Comment (Keerthu94) [inline src/header/header.component.js]: space after arrow

File: src/header/header.component.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './header.css';
```
Comment (Keerthu94) [inline src/header/header.component.js]: Why dont you use css modules?

File: src/header/header.component.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      projectNameClassName.push("close");
+      navBarClassName.push("open");
+  }
+  return(
+    <div className="header">
+      <div className="side-drawer">
+        <div className="logo-content">
+          <Assignment style={{ fontSize: "190%" }} className="thervu-logo" /> 
+        </div> 
+        <div className={projectNameClassName.join(" ")}>
```
Comment (Keerthu94) [inline src/header/header.component.js]: Use single quotes when using js

**Review by Keerthu94 — CHANGES_REQUESTED** [review]

File: src/header/header.component.js (hunk 2 above)
Comment (muruganvenkatesan) [reply src/header/header.component.js]: I had a thought of changing class name so i used let, i will use const here i'm pushing values to assigned object(array) not reassign object(array).

File: src/header/header.component.js (hunk 3 above)
Comment (muruganvenkatesan) [reply src/header/header.component.js]: okay I'll correct

File: package.json (hunk 1 above)
Comment (muruganvenkatesan) [reply package.json:7]: Yes, material ui core is needed for icons

File: src/header/header.component.js (hunk 4 above)
Comment (muruganvenkatesan) [reply src/header/header.component.js]: When i run eject command i got only two webpack config files in my config folder , As per the max and others instructions have to changes for modules in webpack.config.dev.js and webpack.config.prod.js but i got only one js file (webpack.config.js) so i didn't  change modules and use.

File: src/header/header.component.js (hunk 5 above)
Comment (muruganvenkatesan) [reply src/header/header.component.js]: okay, I will change

File: package.json (hunk 1 above)
Comment (Keerthu94) [reply package.json:7]: ok

File: src/App.js [hunk 6]
```diff
@@ -1,26 +1,30 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      isCloseSideDrawer : false
+    }
+  }
+  
+  onChangeSideDrawer = () => {
+    this.setState(state => ({
+      isCloseSideDrawer : !state.isCloseSideDrawer
```
Comment (Keerthu94) [inline src/App.js]: name it as prevState

File: src/App.js [hunk 7]
```diff
@@ -1,26 +1,30 @@
...
+    }));
+  }
+
+  render = () => {
+    return (
+      <div className={classes['app']}>
+       <Header 
+         isCloseSideDrawer={this.state.isCloseSideDrawer} 
+         onChangeSideDrawer={this.onChangeSideDrawer} />
```
Comment (Keerthu94) [inline src/App.js]: FInd out which is better, binding or using arrow functions

Comment (Keerthu94) [inline src/App.js]: change onChangeSideDrawer to onToggleSideDrawer

**Review by Keerthu94 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 6 above)
Comment (muruganvenkatesan) [reply src/App.js]: Ok I'll change

**Review by Keerthu94 — APPROVED** [review]

**Pull request merged**